docs/reference/run.md

```diff
@@ -617,7 +617,8 @@ and require killing system processes to free memory.
 By default, a container's kernel can swap out a percentage of anonymous pages.
 To set this percentage for a container, specify a `--memory-swappiness` value
 between 0 and 100. A value of 0 turns off anonymous page swapping. A value of
-100 sets all anonymous pages as swappable.
+100 sets all anonymous pages as swappable. By default, if you are not using
+`--memory-swapiness`, memory swappiness value will be inherited from parent.
```

Comment:
sorry but since we haven't merged yet let's fix the typo in the flag-name: `--memory-swappiness`.  And let's add the word "from **the** parent." at the end of the sentence to read a little better.